api_spec.yaml

```diff
@@ -54,32 +41,48 @@ paths:
                                       properties:
                                           status:
                                               type: string
-                                              default: "INVALID_CREDENTIALS_ERROR"
-                                          message:
+                                              default: "OK"
+                                          sessionId:
                                               type: string
+                                              example: "5ebe4237-308f-47a8-b823-344b5372ee8f"
                                     - type: object
                                       properties:
                                           status:
                                               type: string
-                                              default: "USER_SUSPENDED_ERROR"
+                                              default: "INVALID_CREDENTIALS_ERROR"
                                           message:
                                               type: string
                                     - type: object
                                       properties:
                                           status:
                                               type: string
-                                              default: "USER_LIMIT_REACHED_ERROR"
+                                              default: "USER_SUSPENDED_ERROR"
                                           message:
                                               type: string
```

Comment:
why changes to the spec in this pr?

Reply:
Just correcting the spec to reflect the final version of the dashboard APIs